src/features/MovieDetails/index.js

```diff
@@ -1,12 +1,32 @@
 import { MovieDetailsWrapper } from "./styled";
 import MovieTopInfo from "./MovieTopInfo";
 import MoviePeople from "./MoviePeople";
+import { useDispatch, useSelector } from "react-redux";
+import { getMovieId, selectDetails, selectStatus } from "./movieDetailsSlice";
+import { useEffect } from "react";
+import { useParams } from "react-router-dom/cjs/react-router-dom.min";
+import { imageUrl } from "../../common/imageUrl";
 
 const MovieDetails = () => {
+  const status = useSelector(selectStatus);
+  const dispatch = useDispatch();
+  const { id } = useParams();
+  const { title, backdrop_path, vote_average, vote_count } =
+    useSelector(selectDetails) ?? {};
+
+  useEffect(() => {
+    dispatch(getMovieId(id));
+  }, [status, dispatch, id]);
+
```

Comment:
There will probably be a need to add another useEffect to clear the API id in future updates.

Reply:
For now, everything displays correctly, so I don't think it's necessary. If there's a need to add a cleanup function in the future, we can always add it then